app/controllers/groups.js

```diff
@@ -0,0 +1,107 @@
+const Groups = require('../models/groups');
+const Clients = require('../models/clients');
+
+module.exports = {
+  index(req, res) {
+    let status = req.query.status === "deleted" ? false : true;
+
+    Groups.findByUser(req.user.cmid, status)
+    .then((groups) => {
+      res.render("groups", {
+        hub: {
+          tab: "groups",
+          sel: status ? "current" : "deleted"
+        },
+        groups: groups
+      });
+    }).catch(res.error500);
+  },
+  new(req, res) {
```

Comment:
I think this is my fault, but it seems we're pretty consistent about leaving a line of whitespace above and below function definitions in controllers. That's great, let's keep doing that.